test/public/runs/overview.test.js

```diff
@@ -1063,28 +1063,36 @@ module.exports = () => {
         await goToPage(page, 'run-overview');
 
         const downloadPath = path.resolve('./download');
+
         // Check accessibility on frontend
         const client = await page.target().createCDPSession();
         await client .send('Page.setDownloadBehavior', {
             behavior: 'allow',
             downloadPath: downloadPath,
         });
 
+        // Open modal in order to prepare to page runs for export
+        await page.$eval(EXPORT_RUNS_TRIGGER_SELECTOR, (button) => button.click());
+        await page.waitForTimeout(100);
```

Comment:
waitForTimeout is deprecated and will be deleted in the next pupeteer major (here: https://github.com/AliceO2Group/Bookkeeping/pull/1387)
Could you try to use waitForSelector instead ?

Reply:
done